_oasis

```diff
@@ -56,7 +56,7 @@ Library xenopsd
     Xenops_server_plugin,
     Xenops_server_skeleton,
     Scheduler,
-    Path,
+    Xpath,
```

Comment:
Hmm, it seems two other components also have similar modules:
```
xc/xc_path.ml
xl/xl_path.ml
```
It's unfortunate that more and more modules have to be renamed as OCaml gains new ones.

I'm not sure what this module is responsible for. Maybe `Paths` or `Resources` or `Resource`? Although then we would also have to rename the corresponding modules in `xc/` and `xl/`.